doc/mf6io/mf6ivar/mf6ivar.py

```diff
@@ -116,21 +116,6 @@
 #   output control rewritten entirely, and implemented in the code
 #
 
-# DEFINITION FILE KEYWORDS
```

Comment:
the [DFN spec](https://modflow6.readthedocs.io/en/latest/_dev/dfn.html) is the source of truth here